lib/nimble_template/addons/faker.ex

```diff
@@ -0,0 +1,14 @@
+defmodule NimbleTemplate.Addons.Faker do
+  @moduledoc false
+
+  use NimbleTemplate.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    Generator.inject_mix_dependency(
+      {:faker, latest_package_version(:faker), [only: [:dev, :test], runtime: false]}
```

Comment:
Maybe 

```suggestion
      {:faker, latest_package_version(:faker), only: [:dev, :test], runtime: false}
```

to make it consistent with other addons 💡

Reply:
Updated in 9a2c0eff70f1ebfdbee9c1413f101c3ca471efa6 👍